src/components/DashboardHome/Dashboard.js

```diff
@@ -15,40 +15,39 @@ import {
   TwitterIcon,
   LinkedinIcon,
   WhatsappIcon,
-  EmailIcon
-} from 'react-share'
+  EmailIcon,
+} from "react-share";
+import Navbar from "../Navbar";
 
 const Dashboard = () => {
-  const dispatch = useDispatch()
+  const dispatch = useDispatch();
   useEffect(() => {
-    dispatch(creators.getCurrentUser())
-  }, [dispatch])
-  const user = useSelector(state => state.user.user)
-  const url = `${window.location.origin.toString()}/store/${user &&
-    user.storeName &&
-    user.storeName
-      .toLowerCase()
-      .split(' ')
-      .join('-')}-${user && user._id}`
-  const storeLogo = user.imageUrl ? user.imageUrl : NoLogo
+    dispatch(creators.getCurrentUser());
+  }, [dispatch]);
+  const user = useSelector((state) => state.user.user);
+  const url = `${window.location.origin.toString()}/store/${
+    user && user.storeName && user.storeName.toLowerCase().split(" ").join("-")
+  }-${user && user._id}`;
+  const storeLogo = user.imageUrl ? user.imageUrl : NoLogo;
   const copied = () => {
-    message.success('url copied successfully')
-  }
+    message.success("url copied successfully");
+  };
   return (
     <div className='mainDiv'>
+      {/* <Navbar /> */}
```

Comment:
How come this is commented out? Is this not suppose to be here?

Reply:
Thank you for your review. All changes have been made.